terraform/aws/components/gp2gp/locals_env_variables.tf

```diff
@@ -44,9 +44,13 @@ locals {
       name = "GP2GP_GPC_GET_DOCUMENT_ENDPOINT"
       value = var.gp2gp_gpc_get_document_endpoint
     },
+    {
+      name = "GP2GP_EXTRACT_CACHE_BUCKET"
```

Comment:
The adaptor requires two environment variables: https://github.com/nhsconnect/integration-adaptor-gp2gp#file-storage-configuration-options

GP2GP_STORAGE_TYPE=S3
GP2GP_STORAGE_CONTAINER_NAME=bucket.name

Reply:
Incorporated suggested change.